README.md

```diff
@@ -108,6 +109,7 @@ Futher detail [here (staff only)](https://docs.google.com/document/d/1muN4pFqRhL
       <ul>
         <li>Contributes ideas for solutions and process improvements in team discussions, and is able to build on the ideas of others.</li>
         <li>Actively exploring new approaches in their own work.</li>
+        <li>Uses relevant data or metrics as an input to solutions.</li>
```

Comment:
Did we say we want to label these as "coming soon/coming Nov 2023" for now? I liked that idea

Reply:
I liked that idea too.

Reply:
sounds good - will it be added as part of the PR?

Reply:
Latest commit is my attempt at this - does it seem ok and clear? Edit suggestions welcome